README.md

```diff
@@ -1,107 +1,17 @@
-# fd-vue-webapp
```

Comment:
Why you modified this README?

Reply:
The project was created using **quasar-cli**, that's why I have **a/**, so details will be port over.